openapi-spec/swagger.yaml

```diff
@@ -2445,6 +2504,8 @@ components:
               type: string
               readOnly: true
             name:
```

Comment:
tenantId, userId are not required in request body

Reply:
Done